packages/insomnia-inso/tsconfig.build.json

```diff
@@ -0,0 +1,28 @@
+{
+  "extends": "../../tsconfig.base.json",
+  "compilerOptions": {
+    // "esModuleInterop": true,
```

Comment:
I assume this isnt an issue because this doesnt appear to be a real json file, but just wanted to check on these "commented out" lines.

Reply:
see: https://kongstrong.slack.com/archives/GPARXT60G/p1616428970018300

the resolution of which is that I'm removing it for now, anyway, just to keep this work moving along.